### PR TITLE
Add Heal command

### PR DESCRIPTION
**Pull request by macha-dev**
Added /heal command, which does the same thing as final heal.

### CODE REVIEW

**Comment by BelKed** [comment]
I suggest adding also ability to heal a specific player. Such as: `/heal <player>`.

**Comment by macha-dev** [comment]
> I suggest adding also ability to heal a specific player. Such as: `/heal <player>`.

Added in most recent commit, along with permissions for the command.

File: src/main/java/com/gmail/val59000mc/commands/HealCommandExecutor.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    public HealCommandExecutor(PlayerManager playerManager, GameManager gameManager) {
+        this.playerManager = playerManager;
+        this.gameManager = gameManager;
+    }
+
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        if (sender.hasPermission("uhc-core.commands.heal")) {
+            if (args.length == 0) { // heal all players
```
Comment (PercyDan54) [inline src/main/java/com/gmail/val59000mc/commands/HealCommandExecutor.java:26]: I prefer healing the sender instead, as Essentials does, and heal all players by /heal *

Comment (MadeByIToncek) [reply src/main/java/com/gmail/val59000mc/commands/HealCommandExecutor.java:26]: This repo is no longer maintained here, please submit it on https://gitlab.com/uhccore/uhccore

**Pull request closed without merging**